README.md

```diff
@@ -43,23 +43,21 @@ make generate
 
 ## Development Cycle
 
-In order to keep the `ccip-develop` branch in working condition, we need to make sure the integration test
-[written in the CCIP repo](https://github.com/smartcontractkit/ccip/blob/03ae3bbed0e6020be5fa9be26d03af21f152d7dc/core/capabilities/ccip/ccip_integration_tests/ocr3_node_test.go#L37)
+In order to keep the `main` branch in working condition, we need to make sure the integration tests
+[written in the CCIP repo](https://github.com/smartcontractkit/chainlink/blob/340a6bfdf54745dd1c6d9f322d9c9515c97060bb/deployment/ccip/changeset/initial_deploy_test.go#L19)
```

Comment:
Wait, we run only this single test? It doesn't seem correct, does it? I think as part of this CI action we run every e2e/integration test for ccip 1.6 we have in the chainlink repository. Especially now, when putting more effort to work on these

Reply:
Yeah we need to rethink these, maybe as a matrix so we can parallelize